.drone.yml

```diff
@@ -0,0 +1,19 @@
+kind: pipeline
+name: default
+
+steps:
+- name: trigger  
+  image: plugins/downstream
+  fork: true
+  settings:
+    server: https://zkopru.sifnoc.net
```

Comment:
should we use our internal server instead?

Reply:
my fault, fixed at 5c16e6d